### DISCUSSION TITLE
Finding a date for the next ESMValTool Workshop

### DISCUSSION THREAD

**Discussion by hb326** [ESMValGroup/Community#19] [Meetings]
Hey all @ESMValGroup/esmvaltool-developmentteam,
we are trying to find dates for the next ESMValTool workshop in late spring/early summer. 
Fill in the following poll if you are interested in joining us:
https://terminplaner4.dfn.de/eJT7MvVewvvgIJHZ

At the moment we are not sure if we could have an in-person meeting at DLR in May/June. If we can't we will have one more virtual meeting during the time that works for most according to the poll. If we can, we plan on having the meeting at DLR as an in-person meeting with a hybrid option for people who cannot or do not want to trave.

Please fill in the poll by **Friday, March 25, 2022.**

**Comment by bouweandela** [comment]
Looking forward to the next workshop!

@hb326 Would it be OK to transfer this discussion to the [Community](https://github.com/ESMValGroup/Community/discussions) repository?

**Reply by hb326** [reply]
> Looking forward to the next workshop!
> 
> @hb326 Would it be OK to transfer this discussion to the [Community](https://github.com/ESMValGroup/Community/discussions) repository?

@bouweandela: sure!

**Comment by hb326** [comment]
Hey @ESMValGroup/esmvaltool-developmentteam,
this is a gentle reminder that the poll for finding a date for the next (hopefully in person) ESMValTool workshop is open until the end of this week!
So please check out the suggested dates and indicate your availability! 

The workshop is planned to be taking place in beautiful Bavarian Oberpfaffenhofen.  :)

**Comment by hb326** [comment]
Hey @ESMValGroup/esmvaltool-developmentteam,
according to the poll, the most people could attend the workshop from **June 13-15 (Monday - Wednesday)**.
So we will finalize this date now, and hope that we can indeed have the workshop in person!
